index.html

```diff
@@ -1,194 +1,198 @@
-            <li>Web Development</li>
-            <img src="images/light.png" alt="light-bulb" class="bulb">
-        </ul>
+            <ul class="lang">
+                <li id="title">Languages</li>
+                <li>JavaScript (ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <img src="images/ic_languages_mob.svg" alt="testimage" class="hashtag">
+            </ul>
+            <ul class="fram">
+                <li id="title">Frameworks</li>
+                <li>React.js</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <img src="images/Rectangle 18.png" alt="rectangle18" class="rectangle18">
+                <img src="images/Rectangle 19.png" alt="rectangle19" class="rectangle19">
+            </ul>
+            <ul class="skills">
+                <li id="title">Skills</li>
+                <li>Database Management</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Version Control</li>
+                <li>Web Development</li>
+                <img src="images/light.png" alt="light-bulb" class="bulb">
+            </ul>
 
           <div class="get-contact">
             <img src="images/Shape (1).png" alt="image4" class="image4">
             <img src="images/Shape (2).png" alt="image5" class="image5">
+
+            <div class="contact-page">
             <h1>Get Started</h1>
-            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</p>
-            <form class="contact-form">
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <form action="https://formspree.io/f/{form_id}" method="post" class="contact-form">
             <div>
               <label id="name-label">Your name</label>
               <br>
-              <input id="name"  type="text" placeholder="Ricardo"/>
+              <input name="name" id="name" type="text" placeholder="Ricardo" maxlength="30" required/>
             </div>
             <div>
               <label id="email-label">Your email</label>
               <br>
-              <input id="email" type="email" placeholder="rj_egas@..."/>
+              <input name="email" id="email" type="email" placeholder="rj_egas@..." required/>
             </div>
             <div>
               <label>Your message</label>
               <br>
-              <textarea name="comments" id="comment-box" cols="30" rows="10" placeholder="Enter your message here..."></textarea>
+              <textarea name="comments" id="comment-box" cols="30" rows="10" placeholder="Enter your message here..." maxlength="500" required></textarea>
             </div>
             <div>
               <button type="button" class="Collaboration">Start Collaboration</button>
             </div>
```

Comment:
- [ ] While you make a great effort 😃, I see that you forgot to add the type submit instead of the button as when I click on it it is not work, so please try to make it type submit to let them from working 

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
| --- |
| ![Recording 2023-04-05 at 12 40 56](https://user-images.githubusercontent.com/83360636/230057712-129f4698-024a-49f8-84b1-cf8aee5fdaa3.gif)   |